### PR TITLE
Removed "old" school targets import

### PR DESCRIPTION
**Pull request by STRUDSO**
Wanted to enable the use of paket for one of my projects, but stumbled into install.ps1 not being executed, so what do you think of this pull request.

### CODE REVIEW

**Comment by aprooks** [comment]
Would be awesome if it helps. I'm also eager to migrate my solution with 15+ projects to paket

File: build.ps1 [hunk 1]
```diff
@@ -59,9 +59,11 @@ task Package -depends Build {
...
     mkdir .\build\content\net40
     mkdir .\build\content\netcore45
-    mkdir .\build\tools
+    mkdir .\build\tools	
     dir -recurse .\source\OctoPack.Tasks\bin\$configuration | copy -destination build\tools -Force
     dir -recurse .\source\tools | copy -destination build\tools -Force
+	mkdir .\build\build
+	dir -recurse .\source\build | copy -destination build\build -Force
     Copy-Item .\source\OctoPack.nuspec .\build 
     Copy-Item .\license.txt .\build
```
Comment (josteink) [inline build.ps1]: Inconsistent tab/space-width compared to rest of code.

**Comment by aprooks** [comment]
@josteink if @STRUDSO  doesn't respond in couple of days I'll update it. Though I have no idea if it works and how to test it.

File: build.ps1 (hunk 1 above)
Comment (STRUDSO) [inline build.ps1]: image
I guess that file is not consistent with itself ;-)

**Comment by STRUDSO** [comment]
they way i tested it was:
importing psake:
Import-Module .\tools\psake\psake.psm1
execute psake:
Invoke-psake .\build.ps1

the  you can invoke nuget within visual studio and point to:
install-package OctoPack - source {REPLACE}\build\OctoPack.3.0.0-alpha.nupkg

**Comment by STRUDSO** [comment]
let me know if there is more i can do?

**Comment by asizikov** [comment]
@PaulStovell  Is there a reason for this PR to be abandoned? 
Currently I'm trying to migrate my projects to NuGet 3.3 which doesn't execute `Install.ps1` file. Instead of old way all the targets should be picked up from `/build` folder (as it's proposed in this Pull Request`).

**Comment by robpearson** [comment]
HI @STRUDSO.  I'm very sorry that we've ignored this PR for so long.  Previously, we weren't focused on paket support so we didn't really give this much attention.  @tothegills and I are revisiting this and #77 to see how we add support for newer frameworks but also maintain compatibility with existing clients.  We hope to have more information available shortly.  Thanks again!

**Comment by rasmus** [comment]
Any update on this? Manually editing csproj files to add the OctoPack taget is a bit of an annoyance. We are constantly creating new micro services, so it feels like I'm doing it on a daily basis.

**Comment by robpearson** [comment]
Just a quick update.  I'm going to close this issue now and we're going to finish/test this in #77 as it addresses the same issue.  We're going to work w/ @asizikov to get his updated PR tested and merged.

**Pull request closed without merging**